### PR TITLE
Unclear markcommand ug

### PR DESCRIPTION
**Pull request by Gabau**

### CODE REVIEW

**Review by mweeruien — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -244,8 +248,13 @@ Removes the period that was marked by the `mark` command.
 
 The format of the input date is in: `YYYY-MM-DD`.
 
+* If no date input is provided, current week is taken as default.
+* If only one date input is provided, the shift added is to the next date that the shift is at. For instance,
+  if the shift is on a 1/10/2021, a monday, with `da/2021-10-27` as input, the shift will be added to 1/10/2021.
```
Comment (fullfatwasabi) [inline docs/UserGuide.md:253]:
```suggestion
* If no date input is provided, the next occurrence of that shift is assumed.
* If only one date input is provided, the next occurrence of that shift, after the provided date is unmarked. For instance,
  if the shift is on Monday 1/11/2021, with `da/2021-10-27` as the date input, the shift on 1/10/2021 would be unmarked.
```

**Review by fullfatwasabi — CHANGES_REQUESTED** [review]
Otherwise, LGTM!

**Review by fullfatwasabi — APPROVED** [review]
LGTM!

**Pull request merged**